### PR TITLE
Support logical_range_filter command

### PR DESCRIPTION
**Pull request by cosmo0920**
#3

### CODE REVIEW

File: lib/groonga/command/logical-range-filter.rb [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+      end
+
+      # @return [String] `order` parameter value.
+      #
+      # @since 1.1.1
+      def order
+        self[:order]
+      end
+
+      # @return [String] `offset` parameter value.
```
Comment (kou) [inline lib/groonga/command/logical-range-filter.rb]: It's not `String`.

File: lib/groonga/command/logical-range-filter.rb [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+      #
+      # @since 1.1.1
+      def offset
+        value = self[:offset]
+        value = value.to_i unless value.nil?
+        value
+      end
+
+      # @return [String] `limit` parameter value.
```
Comment (kou) [inline lib/groonga/command/logical-range-filter.rb]: It's not `String`.

**Comment by cosmo0920** [comment]
I've fixed it!

**Comment by kou** [comment]
It seems that a test for `order` is missing.

**Comment by cosmo0920** [comment]
I added missing test for `order` and fixed typo.

File: test/command/test-logical-range-filter.rb [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+  class ConstructorTest < self
+    def test_ordered_arguments
+      logical_table  = "Logs"
+      shard_key      = "timestamp",
+      min            = "2015-02-12 00:00:00"
+      min_border     = "include"
+      max            = "2015-02-13 00:00:00"
+      max_border     = "exclude"
+      order          = "ascending"
+      filter         = "value == 10"
```
Comment (kou) [inline test/command/test-logical-range-filter.rb]: Use the same order of `ordered_arguments`.

**Comment by kou** [comment]
Thanks. Merged.

**Comment by cosmo0920** [comment]
Thanks for merging!

**Pull request merged**